addons/website_sale/views/templates.xml

```diff
@@ -65,14 +65,14 @@
     </template>
 
     <template id="search" name="Search Box">
-        <form t-att-action="keep('/shop'+ ('/category/'+slug(category)) if category else None, search=0)" method="get" t-att-class="_classes">
-            <t t-if="attrib_values">
-                <t t-foreach="attrib_values" t-as="a">
-                    <input type="hidden" name="attrib" t-att-value="'%s-%s' % (a[0], a[1])" />
-                </t>
+        <t t-if="attrib_values">
+            <t t-foreach="attrib_values" t-as="a">
+                <input type="hidden" name="attrib" t-att-value="'%s-%s' % (a[0], a[1])" />
```

Comment:
Those inputs should still be in the form otherwise they are useless. Put them inside the `t` with the `t-call` and in the form definition, put `<t t-raw="0"/>` which will render what was put in the `t` with the `t-call`